conf/application.conf

```diff
@@ -104,3 +110,6 @@ crypto {
   isEnabled = true
 }
 
+features {
+  email-verification-integration: false
```

Comment:
@matthew-goh - I would maybe flip this toggle so its always fully integrated by default and will toggle on the stubbed alternative? Less likely to end up causing production issues if the config becomes incorrect or get neglected by another team? 

So `_email-verification-stub: true_`